### ISSUE TITLE
bug: Could not download update

### ISSUE THREAD

**Issue by Dor-bl** [appium/appium-inspector#1648]
### Is this an issue specifically with Appium Inspector?

- [X] I have verified that my issue does _not_ occur with Appium, and should be investigated as an Appium Inspector issue

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

1. Click on File
2. Check for updates...
3. version 2024.8.2 is available 
4. Install now
5. error is thrown: Failed to download the update ( see image)
![image](https://github.com/user-attachments/assets/54fb0461-875c-4ae9-b541-9765f51e6ed9)

### Expected Behavior

Update process should finish successfully 

### Operating System

Windows

### Appium Inspector Version

2024.6.1

### Appium Version

2.11.3

**Comment by eglitise** [comment]
Yeah, unfortunately this is an issue :( It also doesn't work on macOS. Not really sure why the certificate expired, but I assume it can be fixed by adding a new one.

**Comment by jlipps** [comment]
Do we need the OpenJSF to generate new certs for us?

**Comment by eglitise** [comment]
I think it's worth a try. There's also a chance it might fix the long-standing issue with Windows/macOS throwing a warning when installing the app.

**Comment by ryanaslett** [comment]
Hi there, reaching out from the Linux Foundation IT on behalf of OpenJSF to see if we can help get these certificates sorted out.

@jlipps can we schedule a time to get all the right certs created and deployed?

**Comment by hmartinez82** [comment]
@eglitise  You don't have to add one. It already has two (one SHA1 and one SHA256). You'd normally have two if there's old code that only recognizes the old certificate. But since everything is broken here, it would be nice to have a single SHA256+ certificate that it's actually recognized  by the OSes

Both the existing certificates are using a completely unknown CA (Certificate Authority) by the OS 

![Image](https://github.com/user-attachments/assets/5715e30a-3feb-43b5-b8f5-1ef808fd5366)

**Comment by ryanaslett** [comment]
NodeJS is now successfully using Azure Trusted Signing via the OpenJS foundations' Azure account. Feel free to reach out if you need help getting this all working.
